zcash_primitives/src/keys.rs

```diff
@@ -180,6 +180,30 @@ impl FullViewingKey {
     }
 }
 
+#[cfg(any(test, feature = "test-dependencies"))]
+pub mod testing {
+    use proptest::collection::vec;
+    use proptest::prelude::*;
```

Comment:
I find it harder to understand code that uses the `use  A::...*` pattern.
Is there any chance you'd be willing to provide the bound Items explicitly?

Reply:
I have to admit that here I was just cargo-culting the proptest examples. I could try to figure out everything that's being used.